src/js/Constants.js

```diff
@@ -59,11 +59,15 @@ var Constants = {
   // The datastore limit is 1 MiB, which we roughly approximate to 1 million characters.
   APPENGINE_SAVE_LIMIT : 1 * 1024 * 1024,
 
+  // Message displayed when an action will lead to erase the current animation.
+  CONFIRM_OVERWRITE: 'This will replace your current animation, are you sure you want to continue?',
+
   // SERVICE URLS
   APPENGINE_SAVE_URL : 'save',
-  IMAGE_SERVICE_UPLOAD_URL : 'http://piskel-imgstore-b.appspot.com/__/upload',
-  IMAGE_SERVICE_GET_URL : 'http://piskel-imgstore-b.appspot.com/img/',
 
   // Code.org - hide or show all layer controls from the UI
-  ENABLE_MULTIPLE_LAYERS: false
+  ENABLE_MULTIPLE_LAYERS: false,
+
+  IMAGE_SERVICE_UPLOAD_URL : '{{protocol}}://piskel-imgstore-b.appspot.com/__/upload',
+  IMAGE_SERVICE_GET_URL : '{{protocol}}://piskel-imgstore-b.appspot.com/img/'
```

Comment:
Nit: Can we move these two constants back up to their original position and keep the Code.org-specific constant at the very end of the file?